sklearn/neural_network/multilayer_perceptron.py

```diff
@@ -273,18 +274,14 @@ def _initialize(self, y, layer_units):
         # Output for binary class and multi-label
         else:
             self.out_activation_ = 'logistic'
-            if self.loss == 'log_loss':
-                self.loss = 'binary_log_loss'
```

Comment:
We don't want to change the value of `self.loss`, and `self.loss` is actually only used once in `_backprop` at line 225 in the original code and 223 in the modified version, so I moved the logic of using 'binary_log_loss' instead of 'log_loss' for multilabel case to line 224-225